File: 8 Kyu/Scala/CalculateBMI.scala
/**
 * https://www.codewars.com/kata/57a429e253ba3381850000fb
 */
object CalculateBMI {
  def bmi(weight: Int, height: Double): String = {
    val bmi = weight / (height * height)
    bmi match {
      case x if x <= 18.5 => "Underweight"
      case x if x <= 25.0 => "Normal"
      case x if x <= 30.0 => "Overweight"
      case _ => "Obese"
    }
  }
}

File: 8 Kyu/Haskell/calculate_bmi.hs
-- https://www.codewars.com/kata/57a429e253ba3381850000fb
module BodyMassIndex where

bmi :: Float -> Float -> String
bmi weight height
  | weight / height ^ 2 <= 18.5 = "Underweight"
  | weight / height ^ 2 <= 25.0 = "Normal"
  | weight / height ^ 2 <= 30.0 = "Overweight"
  | otherwise = "Obese"